README.md

```diff
@@ -35,3 +35,28 @@ and delete this line:
 
 then run brew install giter8 again.
 
+
+# How to deploy your post
+Merging your PR will update the `master` branch only. Besides `master` there's also `acceptance` and `production` branches.
+
+### Getting your post to the acceptance environment
+`acceptance` will allow you to see how your post looks like in `https://blog.acceptance.lunatech.com/`.
+```
+git checkout acceptance
+git rebase master
+git push origin acceptance
+```
+The blog engine needs to be manually restarted in clever cloud as well. Please ask your colleagues, if you don't know where to find it.
```

Comment:
Well I have second thoughts about that. I was expecting this repo to be private. Being it public and that no approvals are needed to merge PR's, anyone could easily deploy anything in the blog. Wouldn't it make sense to add some security somewhere? Someone can still merge something that would inadvertedly be deployed together with another blog post

Reply:
Aah yes, I mixed this one up with the blog engine itself 😅 

Good point, I'll take a look at the repo settings right away 👍

Reply:
The repo is public, but only people with write access can merge and push. Write access is limited to Lunatechies, so that should be good.

Did just setup branch protection rules for the master branch.

Reply:
Good to know 👍